docs/source/release-notes.rst

```diff
@@ -3,6 +3,149 @@ FiftyOne Release Notes
 
 .. default-role:: code
 
+FiftyOne Enterprise 2.12.0
+--------------------------
+*Released October 17, 2025*
+
+Includes all updates from :ref:`FiftyOne 1.9.0 <release-notes-v1.9.0>`, plus:
+
+Plugins, Operators, and Orchestrators
+
+- Added ability to manually terminate an :ref:`Operator <using-operators>` 
+  while its running in any supported
+  :ref:`orchestrator <enterprise-delegated-orchestrator>`: built-in,
+  Databricks, and Anyscale.
+- Updated expiration logic for Delegated :ref:`Operators <using-operators>`,
+  and introduced `FIFTYONE_DO_EXPIRATION_MINUTES`. Instead of considering an
+  Operator terminated one day after *starting*, we now only kill an Operator if
+  it hasn't *updated* in `FIFTYONE_DO_EXPIRATION_MINUTES` minutes. This enables
+  tighter control over execution, and allows for jobs to run for more than one
+  day.
+- Added connection retry logic to a number of
+  :ref:`orchestrators <enterprise-delegated-orchestrator>`-related requests in
+  order to improve resilience.
+- Improved logging and logging levels when plugin secrets are not provided.
+
+App
+
+- Added a plugin secret `FIFTYONE_ZOO_ALLOWED_MODEL_NAMES` that allows FOE
+  admins to configure which zoo models should be available to users when using
+  builtin App features, such as the
+  :ref:`Embeddings Panel <app-embeddings-panel>` and the
+  :ref:`Data Quality Panel <data-quality>`.
+- Ensure that user-provided brain/eval keys are valid in the
+  :ref:`Embeddings Panel <app-embeddings-panel>` and the
+  :ref:`Model Evaluation Panel <app-model-evaluation-panel>`
+  to prevent runtime errors.
+- Fixed a bug where, after a user first logged in, media in the Grid would fail
+  to load because the user's cloud credentials were delayed.
+
+Core
+
+- Reduced likelihood of timeouts when performing `delete_sample_fields`.
+- Improved performance of various operations on media files that can be
+  executed without downloading the full media file.
+- Added some additional logging in the API to aid in debugging and performance
+  monitoring.
+- Added "User-Agent" to a few requests that were missing it.
+
+Security
+
+- Updated a number of dependencies in order to resolve security
+  vulnerabilities: `axios`, `brace-expansion`, `next`, and `starlette`.
+
+
+
+.. _release-notes-v1.9.0:
+
+FiftyOne 1.9.0
+--------------
+*Released October 17, 2025*
+
+Plugins and Operators
+
+- Added `PipelineOperator`, an
+  :ref:`Operator <using-operators>` that can execute a multi-stage workflow,
+  including stages that support
+  :ref:`distributed execution <enterprise-distributed-execution>`. Added formal
+  `Pipeline` and `PipelineStage` types, as well as `PipelineOperator`, to
+  support pipeline-based workflows.
+  `#6341 <https://github.com/voxel51/fiftyone/pull/6341>`_, 
+  `#6385 <https://github.com/voxel51/fiftyone/pull/6385>`_
```

Comment:
I don't mind it being mentioned but these should not be considered production / used. Their types may change.